src/components/Colors.scss

```diff
@@ -34,4 +34,10 @@
   // Only use these reds in error states
   --BrandPasty: #ffe0de;
   --BrandSunburn: #e11f13;
+
+  // For EPLI
+  --BrandMoss--opaque: #f2f9f6;
+  --BrandDuckegg--translucent: #f2f7ff;
+  --BrandDuckegg--dark: #336cc3;
+  --Gold--translucent: #ffc24d;
```

Comment:
Are these the same names design team is using in Figma?

Reply:
Still trying to confirm these with Kelly. These are placeholder names for now.

Reply:
Kelly and I have decided to hard-code these colors for EPLI for now